src/Symfony/Component/DependencyInjection/EnvVarProcessor.php

```diff
@@ -102,15 +102,19 @@ public function getEnv($prefix, $name, \Closure $getEnv)
             return '' === $default ? null : $this->container->getParameter($default);
         }
 
-        if ('file' === $prefix) {
+        if ('file' === $prefix || 'require' === $prefix) {
             if (!is_scalar($file = $getEnv($name))) {
                 throw new RuntimeException(sprintf('Invalid file name: env var "%s" is non-scalar.', $name));
             }
             if (!file_exists($file)) {
                 throw new EnvNotFoundException(sprintf('File "%s" not found (resolved from "%s").', $file, $name));
             }
 
-            return file_get_contents($file);
+            if ('file' === $prefix) {
+                return file_get_contents($file);
+            } else { // 'require'
+                return require $file;
```

Comment:
what about using `@include` and forcing this to return an array?
the array declaration is missing in the property list at the begining of the class btw
when the file is not found (include return false), we would throw a not found exception to work with the "default" processor

Reply:
Added the declaration at the beginning of the class.

Reply:
We talked about this and agreed that as we already have a `file_exists` check in place, we can keep this as-is and accept all different return types.